frontend/src/app/util/dom.cljs

```diff
@@ -203,3 +203,12 @@
   [b]
   {:pre [(blob? b)]}
   (js/URL.createObjectURL b))
+
+(defn set-css-property [node property value]
+  (.setProperty (.-style node) property value))
```

Comment:
Even if shadow-cljs does not emit warnings on interop calls, it is recommended to hint the JS objects with `^js`  to avoid unexpected exception on production builds.

Reply:
OK